app.js

```diff
@@ -10,35 +10,24 @@ const checkbox = document.querySelectorAll(
 	'#list-container input[type="checkbox"]'
 );
 const inputBox = document.getElementById('input-box');
-const inputAddButton = document.getElementById('add-button');
-const inputList = document.querySelectorAll('li');
 
 const addNewTask = function () {
 	const newTaskFormInput = document.getElementById('input-box').value;
```

Comment:
Your code is getting the element input-box twice. You don't need to get the same element multiple times. Therefore, you can remove the line:

Replace newTaskFormInput with listInput.value in the following line:

```
taskName.textContent = listInput.value;

```